.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```

Comment:
I would preferably not include IDE specific folder and files. For one, there are many IDEs / editors out there and also users could have their specific settings for an IDE.

Reply:
removed